### ISSUE TITLE
Request: text objects for partial words

### ISSUE THREAD

**Issue by dhruvasagar** [tpope/vim-abolish#43]
Hi,

Just putting it out there, it would be nice to have motions &/or text objects for partials of various forms of words, like for {camel}Case, {snake}_case !?

**Comment by jasdeepgosal** [comment]
As a workaround for not having this functionality, I just added this to my `.vimrc.local` file: for the word `campaign` which I use & misspell often:

```
fun! FixTypos()
  " except in vim configuration files
  if &ft =~ 'vim'
    return
  endif
  silent! %S/camap{a,}ign/campaign/g
endfun
autocmd BufWritePre * call FixTypos()
```

**Comment by dhruvasagar** [comment]
@jasdeepgosal I am not sure how this is related to what I asked for.

**Comment by jasdeepgosal** [comment]
Oh sorry, feel free to disregard. That snipped allowed me to replace the misplaced word even if it was inside another word, ex: "camapigns_controller" => "campaigns_controller".

**Comment by dhruvasagar** [comment]
Right, but that's too specific and i'd rather just use `:S` for it

**Comment by dhruvasagar** [comment]
@jasdeepgosal btw, you should read the documentation a bit more carefully, you don't have to use :%S in this context, `:Abolish` already allows you to accomplish that.
